astropy/modeling/functional_models.py

```diff
@@ -1066,10 +1066,14 @@ class Voigt1D(Fittable1DModel):
         plt.show()
     """
 
-    x_0 = Parameter(default=0)
-    amplitude_L = Parameter(default=1)     # noqa: N815
-    fwhm_L = Parameter(default=2/np.pi)    # noqa: N815
-    fwhm_G = Parameter(default=np.log(2))  # noqa: N815
+    x_0 = Parameter(default=0,
+                    description="Position of the peak")
+    amplitude_L = Parameter(default=1,
+                            description="The Lorentzian amplitude") # noqa: N815
+    fwhm_L = Parameter(default=2/np.pi,
+                       description="The Lorentzian full width at half maximum") # noqa: N815
+    fwhm_G = Parameter(default=np.log(2),
+                       description="The Gaussian full width at half maximum") # noqa: N815
```

Comment:
```suggestion
    amplitude_L = Parameter(default=1,     # noqa: N815
                            description="The Lorentzian amplitude")
    fwhm_L = Parameter(default=2/np.pi,    # noqa: N815
                       description="The Lorentzian full width at half maximum")
    fwhm_G = Parameter(default=np.log(2),  # noqa: N815
                       description="The Gaussian full width at half maximum")
```
What I meant in my earlier comment was that the `# noqa: N815` tells the syntax checker to ignore the uppercase in the variable names, so it needs to be on the line where they are defined (it is perhaps a bit perfectionist, since those warnings are not treated as failures in the code style checks, so one could simply ignore them; but it makes it easier to keep track of other syntax violations that might remain). So except for `amplitude_L` everything still fits on a single line, so it would have been easier to keep it that way, but when breaking the line, it needs to be done like above. Also note the 2 blanks minimum space before inline comments.